etcd/go.mod

```diff
@@ -0,0 +1,26 @@
+module github.com/gofiber/storage/etcd
+
+go 1.20
```

Comment:
This should be go 1.19, the run `go mod tidy`

Reply:
Done https://github.com/gofiber/storage/pull/878/commits/2ce3ae15bf98a03f1a501b47a2ae757ef51fc563 .